ocular/src/chain/client/automated_tx_handler.rs

```diff
@@ -1,667 +0,0 @@
-            }
-        }
-
-        // Build messages to delegate
-        let mut msgs: Vec<prost_types::Any> = Vec::new();
-
-        for tx in toml.transaction.iter() {
-            let recipient_account_id = match AccountId::from_str(tx.destination_account) {
-                Ok(res) => res,
-                Err(err) => return Err(AutomatedTxHandlerError::KeyHandling(err.to_string())),
-            };
-
-            msgs.push(
-                MsgSend {
-                    from_address: granter_account_id.clone(),
-                    to_address: recipient_account_id,
-                    amount: vec![
-                        Coin {
-                            denom: toml.sender.denom.parse().expect("Could not parse denom."),
-                            amount: tx.amount.into(),
-                        };
-                        1
-                    ],
-                }
-                .to_any()
-                .expect("Could not serialize msg."),
-            );
-        }
-
-        let grantee_base_account = match self
-            .query_account(grantee_public_info.account.clone().to_string())
-            .await
-        {
-            Ok(res) => res,
-            Err(err) => return Err(AutomatedTxHandlerError::ChainClient(err.to_string())),
-        };
-
-        // Send Msg Exec from grantee
-        let response = match self
-            .execute_authorized_tx(
-                Account {
-                    id: grantee_public_info.account.clone(),
-                    public_key: grantee_public_info.public_key,
-                    private_key: self
-                        .keyring
-                        .get_key(grantee_key_name)
-                        .expect("Could not load grantee key."),
-                },
-                msgs,
-                TxMetadata {
-                    chain_id: self
-                        .config
-                        .chain_id
-                        .parse()
-                        .expect("Could not parse chain id"),
-                    account_number: grantee_base_account.account_number,
-                    sequence_number: grantee_base_account.sequence,
-                    gas_fee: Coin {
-                        denom: toml.sender.denom.parse().expect("Could not parse denom."),
-                        amount: toml.sender.gas_fee.into(),
-                    },
-                    gas_limit: toml.sender.gas_limit,
-                    timeout_height: toml.sender.timeout_height,
-                    memo: toml.sender.memo.to_string(),
-                },
-                None,
-                None,
-            )
-            .await
-        {
-            Ok(res) => res,
-            Err(err) => return Err(AutomatedTxHandlerError::TxBroadcast(err.to_string())),
-        };
-
-        dbg!(response.clone());
-
-        Ok(response)
-    }
-
-    pub async fn execute_batch_transactions(
```

Comment:
Thoughts on keeping this method? While not immediately useful for the airdrop, other users could find this helpful in the future.